share/tools/web_config/themes/none.theme

```diff
@@ -0,0 +1,31 @@
+# name: '(Almost) No Colors'
+# preferred_background: 'black'
```

Comment:
I love this theme - I like to read code on my grayscale e-ink screen, and I sometimes can't read text if the background and foreground are too similar. This fixes those issues, very nice!

Actually the one issue I have in mind is that search matches (`fish_color_search_match`) are combined with other syntax highlighting (instead of overriding them), which can make them hard to read, even on a screen that already supports colors.

So if I type `echo` <kbd>Up</kbd> and the matching `echo` is in command position, the default theme will give it dark blue foreground, and as a search match it has black background background (on a light-themed terminal).
If the search match is an argument, it's better because that's lighter by default.
I wonder if we can/should disable our semantic syntax highlighting inside search matches.
I want to try if this makes things better.